lib/enrichment/src/find_enrichment_table_records.rs

```diff
@@ -54,6 +59,28 @@ impl Function for FindEnrichmentTableRecords {
             .into_owned();
         let condition = arguments.required_object("condition")?;
 
+        let select = arguments
+            .optional_array("select")?
+            .map(|select| {
+                select
+                    .into_iter()
+                    .map(|expr| {
+                        let err = || vrl_core::function::Error::ExpectedStaticExpression {
+                            keyword: "select",
+                            expr: expr.clone(),
+                        };
```

Comment:
Not a huge deal given I imagine this parameter would usually be static, but does this parameter need to be static? It seems like it is something that wouldn't be terribly expensive to evaluate at runtime.

Reply:
Yes, that's a good point. I think it should be dynamic. I can see people wanting to calculate the columns based on some criteria on occasion.